src/pages/events/fall-2021-election.astro

```diff
@@ -0,0 +1,52 @@
+---
+import { Markdown } from 'astro/components';
+import Main from '../../layouts/Main.astro';
+---
+<Main title="Fall 2021 Election">
+	<div class="markdown mb-4">
+		<Markdown>
+			# Fall 2021 Election
+
+			Welcome to the Fall 2021 Election for the MUN Computer Science Society!
+
+			<br />
+
+			We're following a somewhat similar setup to our [previous election](https://muncomputersciencesociety.github.io/election-fall-2020/),
+			with keeping the majority of the interactions online for the sake of safety and convenience; especially for people who have still not arrived into Newfoundland yet.
+
+			## Timeline
+
+			- **Monday, September 20th** applications for candidacy open, closing on **September 27th**.
+			- Campaigning for applicants starts **September 28th**, closing on **October the 3rd**.
+			- Results of election presented (sometime on) **Monday, October the 4th**.
```

Comment:
Can these follow the format of the first point in the timeline (starting with the day that the thing happens, and having a consistent date format)